src/components/tutorial-card/types.ts

```diff
@@ -1,6 +1,12 @@
+import { ApiTutorial } from 'lib/learn-client/api/api-types'
 import { ProductOption } from 'lib/learn-client/types'
 
 export interface TutorialCardProps {
+	/**
+	 * The `id` of the Tutorial relevant to the card.
+	 */
+	id: ApiTutorial['id']
```

Comment:
This would actually be `Tutorial['id]` from this file, https://github.com/hashicorp/dev-portal/blob/main/src/lib/learn-client/types.ts#L11 - since this data is formatted. Just a string so not that it matters much.

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/838/commits/f6816373e5e945e458540c30519570929b66f3cf. Also replaced `TutorialCardPropsWithId` since it's redundant with these overall changes.

Reply:
Cool, I think that change would also apply to other places where you're referencing `ApiTutorial` in this PR.

Reply:
Which places? Many of them are valid uses of that interface

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/838/commits/54013f9f393855d971bede11304c57b6b6da75e2!